bin/addCert.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Source env settings
+. .env
+
+CERTIFICATE_PATH=/usr/local/share/ca-certificates/${LOCAL_DOMAIN}
```

Comment:
Not everyone uses Ubuntu =) this may vary between distors. Maybe we should reflect this in README.